src/scripts/OSFramework/OSUI/Pattern/AccordionItem/AccordionItem.ts

```diff
@@ -139,7 +139,7 @@ namespace OSFramework.OSUI.Patterns.AccordionItem {
 
 		// Method to handle the keyboard interactions
 		private _onToggleCallback(): void {
-			Helper.AsyncInvocation(this._platformEventOnToggle, this.widgetId, this._isOpen);
+			this.triggerPlatformEventplatformCallback(this._platformEventOnToggle, this._isOpen);
```

Comment:
I think this should be renamed to triggerPlatformEventCallback in all occurences